### ISSUE TITLE
Multiple Gridsters on one page?

### ISSUE THREAD

**Issue by timatron** [ducksboard/gridster.js#71]
I haven't been able to get it to work. It seems that the last instantiation of a gridster object overwrites the previous gridsters' settings. I would love to be able to either have multiple on a page dynamically or a gridster WITHIN one li object in another grid. Anyone done this? 

Thanks!

**Comment by activeworx** [comment]
I have added two gridsters on the same page and it works without any problem. Only one is visible at a time... so there is no interaction between them.

**Comment by timatron** [comment]
Were you able to change the sizes of one compared to the other? Or the base
width?

**Comment by activeworx** [comment]
They are both always the same size. One thing you do have to worry about is resizing though. Currently gridster does not support resizing. There is already an issue opened for this.

**Comment by ggravarr** [comment]
I have had some issue with this.  Changing the extend on options to not write into default worked for my purposes.
So line 84 of jquery.gridster.js changes from
      this.options = $.extend(true, defaults, options);
to
      this.options = $.extend(true, {}, defaults, options);
I hope that helps.

**Comment by timatron** [comment]
perfect. just what i was looking for! thanks!

**Comment by timatron** [comment]
Got it working, using namespace and ids. Would love some documentation on this, somewhere, but here is my code snippet.

```

        <div id="grid_big" class="gridster">
          <ul>
            <li data-row="1" data-col="1" data-sizex="2" data-sizey="1"></li>
            <li data-row="3" data-col="1" data-sizex="1" data-sizey="1"></li>

            <li data-row="3" data-col="2" data-sizex="2" data-sizey="1"></li>
            <li data-row="1" data-col="2" data-sizex="2" data-sizey="2"></li>

            <li data-row="1" data-col="4" data-sizex="1" data-sizey="1"></li>
            <li data-row="2" data-col="4" data-sizex="2" data-sizey="1"></li>
            <li data-row="3" data-col="4" data-sizex="1" data-sizey="1"></li>

            <li data-row="1" data-col="5" data-sizex="1" data-sizey="1"></li>
            <li data-row="3" data-col="5" data-sizex="1" data-sizey="1"></li>

            <li data-row="1" data-col="6" data-sizex="1" data-sizey="1"></li>
            <li data-row="2" data-col="6" data-sizex="1" data-sizey="2"></li>
          </ul>
        </div>

      <div id="grid_small" class="gridster">
        <ul>
          <li data-row="1" data-col="1" data-sizex="2" data-sizey="1"></li>
          <li data-row="3" data-col="1" data-sizex="1" data-sizey="1"></li>

          <li data-row="3" data-col="2" data-sizex="2" data-sizey="1"></li>
          <li data-row="1" data-col="2" data-sizex="2" data-sizey="2"></li>

          <li data-row="1" data-col="4" data-sizex="1" data-sizey="1"></li>
          <li data-row="2" data-col="4" data-sizex="2" data-sizey="1"></li>
          <li data-row="3" data-col="4" data-sizex="1" data-sizey="1"></li>

          <li data-row="1" data-col="5" data-sizex="1" data-sizey="1"></li>
          <li data-row="3" data-col="5" data-sizex="1" data-sizey="1"></li>

          <li data-row="1" data-col="6" data-sizex="1" data-sizey="1"></li>
          <li data-row="2" data-col="6" data-sizex="1" data-sizey="2"></li>
        </ul>
      </div>

<script type="text/javascript" src="assets/jquery-1.8.2.min.js"></script>
<script type="text/javascript" src="assets/jquery.gridster.js" charset="utf-8"></script>

<script type="text/javascript">
  var gridster1;
  var gridster2;

  $(function(){


    gridster1 = $("#grid_big > ul").gridster({
          widget_margins: [10, 10],
          widget_base_dimensions: [140, 140],
          min_cols: 6,
          namespace: '#grid_big'
      }).data('#grid_big');

    gridster2 = $("#grid_small > ul").gridster({
          widget_margins: [3, 3],
          widget_base_dimensions: [40, 40],
          min_cols: 6,
          namespace: '#grid_small'
    }).data('#grid_small');

  });
</script>

```

**Comment by yossibens** [comment]
ok it works but how can you enable the 1st one and disable the 2nd ??

**Comment by jim-at-miramontes** [comment]
I have @timatron 's method working; thanks! Followup (on the chance that people are still following this thread): Is there any way to have two regions working independently EXCEPT that it's possible to drag items from one region to the other?  I'm skeptical, but I thought I'd ask...

**Comment by bpbhat77** [comment]
drag between grids posible ???

**Comment by krosenberg** [comment]
I have two grids on the same page using @timatron's method, and everything seems to be working fine. However, when I add a draggable.stop callback function to each, only the last one triggers when i stop dragging a widget on either of the grids. Any idea why this happens?

**Comment by noizwaves** [comment]
+1 vote for @timatron's method to be included in the documentation.

**Comment by danroberts** [comment]
this pull request fixes the event handling errors: #273
